engine/src/Interpreter/interpreter_cpp.cu

```diff
@@ -216,8 +216,573 @@ void calculate_grid(int * min_grid_size, int * block_size, column_index_type max
+//			break;
+//		case cudf::type_id::TIMESTAMP_DAYS:
+//		// TODO percy cudf0.12 by default timestamp for bz is MS but we need to use proper time resolution
+//		// percy this was not commented
+//		//case cudf::type_id::TIMESTAMP_SECONDS:
+//			nv_strings = NVStrings::long2timestamp(static_cast<unsigned long *>(input_col->data),
+//				input_col->size,
+//				NVStrings::days,
+//				"%Y-%m-%d",
+//				input_col->valid);
+//			break;
+//		// TODO percy cudf0.12 by default timestamp for bz is MS but we need to use proper time resolution
+//		case cudf::type_id::TIMESTAMP_MILLISECONDS:
+//			if(input_col->dtype_info.time_unit == TIME_UNIT_ns) {
+//				nv_strings = NVStrings::long2timestamp(static_cast<unsigned long *>(input_col->data),
+//					input_col->size,
+//					NVStrings::ns,
+//					"%Y-%m-%dT%H:%M:%SZ",
+//					input_col->valid);
+//			} else {
+//				nv_strings = NVStrings::long2timestamp(static_cast<unsigned long *>(input_col->data),
+//					input_col->size,
+//					NVStrings::ms,
+//					"%Y-%m-%dT%H:%M:%SZ",
+//					input_col->valid);
+//			}
+//			break;
+//		default: assert(false);
+//		}
+//	}
+
+//	NVCategory * nv_category =
+//		nv_strings ? NVCategory::create_from_strings(*nv_strings) : NVCategory::create_from_array(nullptr, 0);
+//	NVStrings::destroy(nv_strings);
+
+//	gdf_column_cpp new_input_col;
+//	new_input_col.create_gdf_column(nv_category, nv_category->size(), "");
+
+//	return new_input_col;
+}
+
+int insert_string_into_column_nvcategory(const cudf::column_view & col, const std::string & str) {
+	// TODO percy cudf0.12 port to cudf::column and custrings
+//	const char * str_arr[] = {str.c_str()};
+//	NVStrings * temp_string = NVStrings::create_from_array(str_arr, 1);
+//	NVCategory * old_category = static_cast<NVCategory *>(col->dtype_info.category);
+//	NVCategory * new_category = old_category->add_strings(*temp_string);
+//	col->dtype_info.category = new_category;
+
+//	NVStrings::destroy(temp_string);
+//	NVCategory::destroy(old_category);
+
+//	CheckCudaErrors(cudaMemcpyAsync(
+//		col->data, new_category->values_cptr(), sizeof(gdf_nvstring_category) * col->size, cudaMemcpyDeviceToDevice));
+
+//	return new_category->get_value(str.c_str());
+}
+
+std::string like_expression_to_regex_str(const std::string & like_exp) {
+	if(like_exp.empty()) {
+		return like_exp;
+	}
+
+	bool match_start = like_exp[0] != '%';
+	bool match_end = like_exp[like_exp.size() - 1] != '%';
+
+	std::string re = like_exp;
+	static const std::regex any_string_re{R"(([^\\]?|\\{2})%)"};
+	re = std::regex_replace(re, any_string_re, "$1(?:.*?)");
+
+	static const std::regex any_char_re{R"(([^\\]?|\\{2})_)"};
+	re = std::regex_replace(re, any_char_re, "$1(?:.)");
+
+	return (match_start ? "^" : "") + re + (match_end ? "$" : "");
+}
+
+std::unique_ptr<cudf::column> handle_match_regex(const cudf::column_view & input_col, const std::string & re) {
```

Comment:
donde need special sting functions anymore these are going to be handled in interops